docroot/themes/humsci/humsci_basic/src/scss/components/_secondary-nav.scss

```diff
@@ -127,20 +148,22 @@
       background-repeat: no-repeat;
       background-position: center right;
       background-size: hb-calculate-rems(18px);
-      background-image: svg($hb-icon-plus);
-
-      color: darken($su-color-driftwood, 30%);
+      @include hb-icon-background('plus', 'gray-dark');
       height: hb-calculate-rems(18px);
       width: hb-calculate-rems(18px);
       display: block;
       transform: rotate(0deg);
       transition: transform 200ms ease-in-out;
+
+      @include hb-colorful {
+        @include hb-global-color('color', $hb-menu-link-border-active);
```

Comment:
I actually dont love the use of vars like this

it's much easier for me to see "gray-dark" than menu-link-border-active which I'm not sure which color it is

Reply:
just my take!

Reply:
I agree. I refactored the vars out.

Reply:
awesome thanks for doing that